### PR TITLE
Add `destinationPath` option to `.startRecording()`

### LINKED ISSUE #9: Add `destinationPath` option
If anyone needs this, continue with https://github.com/wulkano/aperture-node/pull/7 and address the feedback and include use-cases in the docs.

### PR DESCRIPTION
**Pull request by alectrocute**
Fixes #9

### CODE REVIEW

**Comment by sindresorhus** [comment]
From #9:

> continue with #7 and **address the feedback and include use-cases in the docs.**

**Comment by alectrocute** [comment]
Hi @sindresorhus,

I've added the relevant documentation. Would you like me to go a bit more in-depth?

https://github.com/wulkano/aperture-node/blob/c7341da73ce2def942f486a9aaaf7d3a1f1a6d6b/readme.md

Cheers!

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (sindresorhus) [inline readme.md:1]: Don't do unrelated changes.

File: readme.md [hunk 2]
```diff
@@ -32,7 +33,7 @@ const options = {
   await aperture.startRecording(options);
   await delay(3000);
   console.log(await aperture.stopRecording());
-  //=> '/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/cdf4f7df426c97880f8c10a1600879f7.mp4'
+  //=> '/private/var/folders/3x/jf5977fn79j/T/cdf4f7df426c97880f8c10a1600879f7.mp4'
```
Comment (sindresorhus) [inline readme.md:36]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
The docs should describe why it's beneficial not to use this option, as described in https://github.com/wulkano/aperture-node/pull/7#discussion_r293724113

**Comment by chrisspiegl** [comment]
@alectrocute Do we know what exactly is missing here? I am not very used to working on pull requests and will have to learn the process but I am interested in this Pull Request being picked up again and finished.

Specifically for the use case of storing the temp file on an external NVME SSD to have more storage for ProRes recording.

I'm willing to help but as mentioned, I don't know Pull Request processes that well 🙈.
